tensorboard/uploader/uploader_test.py

```diff
@@ -1411,6 +1424,57 @@ def test_break_at_tensor_point_boundary(self):
             )
         self.assertEqual(total_points_in_result, point_count)
 
+    def test_strip_large_tensors(self):
+        # Generate test data with varying tensor point sizes. Use raw bytes.
+        event_1 = event_pb2.Event(step=1)
+        event_1.summary.value.add(
+            tag="one", tensor=tensor_pb2.TensorProto(tensor_content=b"\x01\x02")
+        )
+        event_1.summary.value.add(
+            tag="two",
+            tensor=tensor_pb2.TensorProto(
+                tensor_content=b"\x01\x02\x03\x04\x05\x06"
+            ),
+        )
+        event_2 = event_pb2.Event(step=2)
+        event_2.summary.value.add(
+            tag="one", tensor=tensor_pb2.TensorProto(tensor_content=b"\x01\x02")
+        )
+        event_2.summary.value.add(
+            tag="two",
+            tensor=tensor_pb2.TensorProto(
+                tensor_content=b"\x01\x02\x03\x04\x05\x06\x07"
+            ),
+        )
+
+        run_proto = self._add_events_and_flush(
+            _apply_compat([event_1, event_2]), max_tensor_point_size=7 + 1
```

Comment:
Why is this "7 + 1"?  Also, since the tensor_content at line 1446 is 7 bytes, one might expect that to pass the filter because 7 < 8.  Presumably the proto wire format overhead is the reason?  Anyway, please add some comments about what is going on here.

Reply:
You're intuition is correct. I've tried to improve the documentation.